cli/index.js

```diff
@@ -770,6 +771,25 @@ RainbowConfig.addOptions(
   ]
 )
 
+RainbowConfig.addOptions(
+  testingCommand
+    .command('get-bridge-on-near-balance')
+    .option(
+      '--near-receiver-account <near_receiver_account>',
+      'The account on NEAR blockchain that received the minted token.'
+    )
+    .description('Gets balance on Near for Rainbow-Bridge Eth on Near Prover.'),
```

Comment:
Why Near Prover?

Reply:
Wrong info related to the old account that was used everywhere.